packages/smooth_app/lib/pages/product/add_basic_details_page.dart

```diff
@@ -168,39 +166,23 @@ class _AddBasicDetailsPageState extends State<AddBasicDetailsPage> {
         ),
       );
 
-  Future<bool> _saveData(LocalDatabase localDatabase) async {
+  Future<bool> _saveData(
```

Comment:
You should probably return a `Product?`.

Reply:
> You should probably return a `Product?`.

✅